stdlib/sys/file.gr

```diff
@@ -818,7 +903,9 @@ export let rec fdStats = (fd: FileDescriptor) => {
 
     let flagsToWasmVal = (flag, i) => {
       let rightsInheriting = WasmI64.load(structPtr, 16n)
-      (rightsInheriting & (1N << WasmI64.extendI32U(WasmI32.fromGrain(i) >> 1n))) > 0N
+      (rightsInheriting &
+      1N << WasmI64.extendI32U(WasmI32.fromGrain(i) >> 1n)) >
+      0N
```

Comment:
This one is interesting. Should we indent the line wrap since it is inside parens?